canonical-web-and-design/ubuntu.com
# Copy update - /ai

## Done

- Updated copy in subheading CTA's and one in the body

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/ai
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please compare the changes against the [copy doc](https://docs.google.com/document/d/1HP9YdHF11yVQ9SLl0VXhLezDZREsr1GHIKNi9wT1Gv0/edit?ts=5fb6b19b#) and accept corrected comments


## Issue / Card

Fixes [#3840](https://github.com/canonical-web-and-design/web-squad/issues/3840)
